### ISSUE TITLE
circleci/jira@1.3.1 is not working correctly with alpine based images

### ISSUE THREAD

**Issue by andormarkus** [CircleCI-Public/jira-connect-orb#64]
Orb version: 1.3.1

### What happened
Based on our investigation the latest (1.3.1) as previous (1.3.0) version of this orb does not work together with alpine based docker images. Tested images: alpine:latest, hashicorp/terraform:latest

Working pipelines
```
jira:
    docker:
        -   image: cimg/base:stable
    steps:
        - jira/notify
```

Not working pipeline
```
jira:
    docker:
        -   image: alpine
    steps:
        - run: apk add --no-cache jq
        - run: apk add --no-cache curl
        - jira/notify
```

Error code
```
Curl failed with code 200. full response below.
{
  "name" : "James Bond",
  "login" : "agent007",
  "id" : "xxxxxx-xxxx-xxxx-xxxx-xxxxxx"
}
````

**Comment by floflock** [comment]
I experienced the same issue.
I use `alpine:3.13` via `google/cloud-sdk:alpine` base image: https://hub.docker.com/layers/google/cloud-sdk/alpine/images/sha256-5c3703f9b58682535099d02094fcabb0fc7a2749dd713a7521ff7f945cd86150?context=explore

Final rotonsole output from `jira/notify`

```
sh: 0: unknown operand
/bin/sh: curl: not found

Exited with code exit status 127
CircleCI received exit code 127
```

**Comment by andormarkus** [comment]
Hi @floflock 

Please add `- run: apk add --no-cache curl` to your pipeline and try again.
Curl is not part of the base alpine linux.

**Comment by floflock** [comment]
Hi @andormarkus 

unfortunately, this was not the issue.

I have added a layer in between to install curl, but it is not working either.

```
FROM google/cloud-sdk:alpine

RUN gcloud components install kubectl --quiet

RUN apk add --update --no-cache \
    curl \
    tar \
    jq \
    ca-certificates

# Install Kustomize
RUN curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" | bash && \
    mv kustomize /usr/local/bin/kustomize && \
    chmod u+x /usr/local/bin/kustomize

COPY scripts/gcloud-authenticate /usr/bin/gcloud-authenticate

RUN chmod +x /usr/bin/gcloud-authenticate
```

**Comment by andormarkus** [comment]
Hi @floflock 

What happens when you add `curl` to the docker image in the pipeline as well? 
Like I did in the description.

**Comment by KyleTryon** [comment]
Hey folks, at the current moment (could be subject to change), we do not actively support alpine. Usage on alpine is typically fairly low in comparison and often difficult to support as mentioned above by @floflock has started to mention, a lot of packages required to run CircleCI jobs are simply not present. Other common packages needed with orbs are utilities such as CURL, also not often available.

This is still _very_ early but we are investigating alternative methods of integrating with Jira/JSM without the need for an orb, given we are releasing webhooks shortly. Once this is in place, it will not be necessary to modify your CI pipeline just to send notifications to Jira.

@andormarkus I agree your suggestion is the best workaround, for Alpine, the best option is to create a custom docker image with all of the required tools. You can see a list of common tools/utilities we often expect to be installed via our CircleCI `cimg/base` docker image: https://github.com/CircleCI-Public/cimg-base/blob/master/Dockerfile.template
